scripts/_autoloads/scenester.gd

```diff
@@ -5,69 +5,141 @@ extends Node
 
 signal loading_progress_changed(progress)
 signal loading_done
+signal scene_has_changed(p_scene: String)
 
-@export var main_menu_scene: RScene
-@export var initial_scene: RScene
+@export_group("Special Scenes")
+@export var main_menu_rscene: RScene
+@export var pause_rscene: RScene
+@export var load_rscene: RScene
+@export var reconnect_rscene: RScene
+@export var initial_rscene: RScene
 @export var should_load_initial_alt: bool = false
+@export var pause_menu: PackedScene = null
```

Comment:
Why do we have a pause menu Packed scena and a pause scene r_scene which should include a packed scene.

Reply:
Nice catch. Old way of doing things, forgot to remove that line. Its removed.